Source/DafnyLanguageServer.Test/Various/ShutdownTest.cs

```diff
@@ -0,0 +1,181 @@
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Reflection;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.Collections.Generic;
+using System.IO.Pipelines;
+using System.Linq;
+using System.Reactive.Concurrency;
+using System.Text;
+using System.Text.Json;
+using System.Threading;
+using Microsoft.CodeAnalysis;
+using OmniSharp.Extensions.JsonRpc;
+using OmniSharp.Extensions.JsonRpc.Client;
+using OmniSharp.Extensions.JsonRpc.Serialization;
+using OmniSharp.Extensions.LanguageServer.Protocol.Client.Capabilities;
+using OmniSharp.Extensions.LanguageServer.Protocol.Models;
+
+namespace Microsoft.Dafny.LanguageServer.IntegrationTest.Various {
+
+  [TestClass]
+  public class ShutdownTest {
+
+    [TestMethod]
```

Comment:
> This looks great. Since it's part of the spec that language server should handle the termination of the parent process, I'm wondering why it's not part of omnisharp.

Yes, I don't understand that either.